mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/resourcecluster/ResourceClusterAkkaImpl.java

```diff
@@ -198,6 +201,23 @@ public CompletableFuture<Ack> disableTaskExecutorsFor(Map<String, String> attrib
                 .toCompletableFuture();
     }
 
+    @Override
+    public CompletableFuture<PagedActiveJobOverview> getActiveJobOverview(
+        Optional<Integer> startingIndex,
+        Optional<Integer> maxSize) {
```

Comment:
nit: I prefer not to use Optionals in method parameters. Here's some context: http://dolszewski.com/java/java-8-optional-use-cases/

Reply:
I read the link and this case actually feels like the exact case where the inputs are already optional<T> from the akka HTTP API for optional params and it doesn't create the extra wrapping on the caller side.

Reply:
That makes sense. let's keep it this way.